tests/test_models.py

```diff
@@ -281,29 +282,27 @@ def on_save(self, init):
 
     model = OnSaveModel()
     assert "foo" not in model.meta._instance
-    model.save(tmp_path/"test.asdf")
+    model.save(tmp_path / "test.asdf")
     assert model.meta.foo == "bar"
 
 
-@pytest.mark.parametrize("ModelType", [DataModel, BasicModel, TableModel, TransformModel])
-def test_garbage_collectable(ModelType, tmp_path):
+@pytest.mark.parametrize("ModelType", [DataModel, BasicModel, TableModel, TransformModel])  # noqa: N803
+def test_garbage_collectable(ModelType, tmp_path):  # noqa: N803
```

Comment:
can this be changed to `model_type`?

Reply:
Since `ModelType` here is a class so I'm in favor of keeping the CamelCase.